### DISCUSSION TITLE
Generate ocean tiles for the whole globe on all zoom levels (with --area)

### DISCUSSION THREAD

**Discussion by adjourn** [onthegomap/planetiler#1056] [Q&A] [answered]
Let's say you only need to generate a map for single country or continent.
Ocean tiles are generated only for the tiles where the country/continent fits into.

Since I need to allow camera pitch and it would be nice to have a little more room for panning than few pixels from boundaries..

Any suggestions how to satisfy the second point?
* Generate selected country/continent with OSM data (✔️)
* Generate Natural Earth ocean tiles for the whole globe for every zoom level (❌)

**Comment by msbarry** [comment, marked as answer]
Could you use an OSM extract but then set `--bounds=world` or something from https://boundingbox.klokantech.com so that the global datasets (water, natural earth, etc) get clipped to a bigger region?

**Reply by adjourn** [reply]
That did indeed work, thank you very much for your help!

**Reply by adjourn** [reply]
@msbarry Apologies for dragging this subject on but quick question.. World seems a bit overkill since I plan to clamp the panning to certain area anyway. I assume `--bounds` also takes a bounding box, if true, what format does it accept?

**Reply by msbarry** [reply]
Yep it takes comma separated string with format `west,south,east,north` like https://boundingbox.klokantech.com/ generates with "CSV" mode. You can also pass in a [".poly" file](https://wiki.openstreetmap.org/wiki/Osmosis/Polygon_Filter_File_Format) to `--polygon` argument that will clip to an irregular shape.
